mmdet/datasets/pipelines/auto_augment.py

```diff
@@ -1,10 +1,40 @@
 import copy
 
+import mmcv
 import numpy as np
 
 from ..builder import PIPELINES
 from .compose import Compose
 
+_MAX_LEVEL = 10
+
+
+def level_to_value(level, max_value):
+    """Map from level to values based on max_value."""
+    return (level / _MAX_LEVEL) * max_value
+
+
+def random_negative(value, random_negative_prob):
```

Comment:
and this

Reply:
Yeah, it will be used for another aug like translate, rotate, etc.